core/src/main/java/io/hyperfoil/core/generators/StringValueProviderBuilder.java

```diff
@@ -0,0 +1,19 @@
+package io.hyperfoil.core.generators;
+
+public class StringValueProviderBuilder<P> extends BaseValueProviderBuilder<P, String> {
```

Comment:
Now I've noticed the class is not used at all - that renders most of the changes above unnecessary.

Reply:
Can I keep the `BaseValueProviderBuilder` or should I rollback?

Reply:
Up to you; if you make it return correct self-type you can keep it. You should probably also inline the buildFromAccess and buildFromConst methods.